src/middlewares/checkUser.js

```diff
@@ -0,0 +1,35 @@
+import models from '../api/models';
+import status from '../helpers/constants/status.codes';
+
+const { User } = models;
+/**
+ * check if user exist
+ *
+ * @export
+ * @class CheckUser
+ */
+class CheckUser {
```

Comment:
@Ramadhan0 I think the naming of this class and file can be better - as it is, we can't understand what the middleware does from its name `CheckUser`